operator/src/main/java/oracle/kubernetes/operator/helpers/DomainPresenceInfo.java

```diff
@@ -361,6 +361,18 @@ public void setServerPodBeingDeleted(String serverName, Boolean isBeingDeleted)
     getSko(serverName).isPodBeingDeleted().set(isBeingDeleted);
   }
 
+  public int getHttpRequestFailureCount(String serverName) {
```

Comment:
Please add Javadoc for these methods. It is not obvious why we have failure counts for http request failures, and docs could save the next coder a lot of time trying to reverse-engineer the code.

Reply:
Added.